dist/site.js

```diff
@@ -32208,7 +32210,7 @@ const addMarkers = (geojson, map, context, writable) => {
   
   pointFeatures.map((d, i) => {
     const marker = new ClickableMarker({
```

Comment:
Someone is totally going to ask about maki next, but I think we can leave off of those for now.

Reply:
+1

Reply:
#721 :)

Reply:
Was it deployed to production? They're still not active for me.